code/__HELPERS/colors.dm

```diff
@@ -17,3 +17,10 @@
 		final_color += copytext(color, digit, digit + 1)
 
 	return final_color
+///Given a hexcolor (#FFFFFF), returns the inverted hexcolor.
+/proc/invert_hex(hexcolor)
+    var/list/old_rgb = hex2rgb(hexcolor)
+    var/inverted_r = 255 - old_rgb[1]
+    var/inverted_g = 255 - old_rgb[2]
```

Comment:
These are spaces, not tabs. this is why the linter is failing on ya

Reply:
As in, before the var/, or between the values?

Reply:
before the var/, easiest way i can demonstrate it:

spaces:
![image](https://user-images.githubusercontent.com/40974010/135727974-49845555-dcf9-4c22-a861-c693bd805d0b.png)

tabs:
![image](https://user-images.githubusercontent.com/40974010/135727937-3261fe99-fa92-465a-98bb-144c0c5a5958.png)

Reply:
Done